src/ert/storage/local_ensemble.py

```diff
@@ -678,14 +881,6 @@ def load_all_gen_kw_data(
         -----
         Any provided keys that are not gen_kw will be ignored.
         """
-        if realization_index is not None:
```

Comment:
No longer used, can be removed from signature

Reply:
On 2nd thought, might have to put it back, summary/gen_kw collector seems to use it in some tests at least. Same applies for the summary loading function